.github/workflows/earthly-test.yml

```diff
@@ -33,13 +34,14 @@ jobs:
         fi
         git checkout -b "$branch" || true
     - name: Download latest earthly
-      run: "sudo /bin/sh -c 'wget https://github.com/earthly/earthly/releases/download/v0.6.26/earthly-linux-amd64 -O /usr/local/bin/earthly && chmod +x /usr/local/bin/earthly'"
+      run: "sudo /bin/sh -c 'wget https://github.com/earthly/earthly/releases/download/v0.6.30/earthly-linux-amd64 -O /usr/local/bin/earthly && chmod +x /usr/local/bin/earthly'"
```

Comment:
> Why can't we use /earthly/releases/latest/...?

With a fixed version the behavior is more predictable. With `/latest` there would always be a chance that something broke because of a new Earthly release.

(I'm in favor of a lockfile for the Conda dependencies too, for the same reason.)

But maybe we can switch to `/latest` when Earthly becomes stable enough, e.g. 1.0.